### ISSUE TITLE
Detecting Trackers Issue

### ISSUE THREAD

**Issue by HonestAbe7** [Hamza417/Inure#40]
When I click on the option to check for trackers it just keeps spinning now endlessly. I think this used to work in a previous version but now it doesn't work anymore.

**Comment by HonestAbe7** [comment]
Ok. Cleared trackers cache with no success.

**Comment by Hamza417** [comment]
@HonestAbe7 Hmm, Which application are you trying to open?

**Comment by HonestAbe7** [comment]
I tried to check QKSMS (even though Warden app says there is a logger it uses but shouldn't have any trackers) for trackers just to test it out under the trackers section. I am noticing if there are no trackers it just doesn't say anything for other apps.

**Comment by Hamza417** [comment]
@HonestAbe7 Yes, it should be showing a warning, or the warning should be in _messages_ popup which might be off and can be accessed by tapping it's icon from the top bar.

**Comment by HonestAbe7** [comment]
Ok. I am enabled showing all classes for trackers but your app crashed while trying to do so and I couldn't send a crash report. The trackers seem to do eventually show but it takes a while. Here is what I enabled.
![Screenshot_20221122-174058](https://user-images.githubusercontent.com/88217236/203436447-e8ace671-907d-47b2-9394-2df0e1756f9f.png)

I just got the report of zero trackers for QKSMS after some time waiting.
